clients/web/src/lib/components/atoms/Tabs/Tabs.svelte

```diff
@@ -0,0 +1,46 @@
+<script lang="ts">
+    import type {TabItems} from "./types";
+    import {Icon} from "@steeze-ui/svelte-icon";
+
+    type T = $$Generic<TabItems>;
+    interface $$Props {
+        tabs: TabItems;
+        activeKey: keyof TabItems;
+    }
+
+    export let tabs: TabItems;
+    export let activeKey: keyof TabItems;
+</script>
+
+<div class="text-sm font-medium text-center border-b text-gray-400 border-gray-700">
```

Comment:
There are some accessibility considerations we need to implement

See
https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/tab_role
https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/tablist_role
https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/tabpanel_role

For implementation, the official `react-tabs` library would be good to follow (there are several `aria` properties we should implement)